plugins/theme/components/commons/page.py

```diff
@@ -34,15 +34,15 @@ def reporting_config(context, slot, payload):
     return render_template(f"common/reporting-config.html", config=payload)
 
 
-def scanners_dast_config(context, slot, payload):
+def applications_scanners_config(context, slot, payload):
```

Comment:
application_scanners is not common component should be placed in theme/components/security/application.py